app/models/vote.rb

```diff
@@ -29,8 +29,12 @@ def self.noes
     where("votes.vote = 'no'")
   end
 
-  def rebellion?
-    !free? && vote != whip_guess
+  def rebellion?(whip=nil)
```

Comment:
Sure. But what do you think about this method overall? I did that to improve the performance, but I hated the end resutlts. I don't have complete understanding about what this method does, so I'm not sure how to improve it. If you have any suggestions, that would be awesome!